var/spack/repos/builtin/packages/papi/package.py

```diff
@@ -46,9 +48,12 @@ class Papi(AutotoolsPackage):
     # and therefore not implemented here
 
     depends_on('lm-sensors', when='+lmsensors')
+    depends_on('cuda', when='+cuda')
+    depends_on('cuda', when='+nvml')
 
     conflicts('%gcc@8:', when='@5.3.0', msg='Requires GCC version less than 8.0')
     conflicts('+sde', when='@:5.9.99999', msg='Software defined events (SDE) added in 6.0.0')
+    conflicts('^cuda', when='@:5.9.99999', msg='CUDA support for versions < 6.0.0 not implemented')
```

Comment:
```suggestion
    conflicts('^cuda', when='@:5', msg='CUDA support for versions < 6.0.0 not implemented')
```
This is more reliable in case a 5.10 release comes out someday

Reply:
We actually dealt with this previously with the "sde" conflict above.  Due to the odd naming convention of the recent releases (6.0.0 and 6.0.0.1), this was the only way to allow any 6.x release and reject any earlier release.  Also, I'm pretty confident that there will not be a 5.10 release of papi.

Reply:
I may be misremembering this though, so let me verify it.

Reply:
You can make the same change to the sde conflict above as well.

Reply:
Yep, done.